packages/basic/src/typography.scss

```
@@ -419,4 +419,23 @@ $pre-border-radius: 0.375rem;
+    &::after {
+      content: attr(data-close);
+      opacity: 0.5;
+    }
+  }
+  .prosekit-dsl-block:not(.prosekit-dsl-head-inside) {
+    margin: 0.5rem 0;
+  }
+  .prosekit-dsl-block .prosekit-dsl-block-display {
+    padding: 0.5rem;
+  }
+  // #endregion DSL
```

Stylelint (17.11.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)